applications/nrf5340_audio/src/modules/dfu_entry.c

```diff
@@ -3,13 +3,17 @@
  *
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
+#define _BSD_SOURCE
```

Comment:
Should add a newline between the license and this #define.
Maybe also add a comment to the code why this define is needed?

Reply:
Add explanation here.

Reply:
I can not see any explanation here in the newest version of the code?